components/picker/stories/picker.test.js

```diff
@@ -9,24 +9,88 @@ export const PickerGroup = Variants({
 	},
 	testData: [
 		{
-			testHeading: "Default",
+			testHeading: "Default, with placeholder",
 		},
 		{
-			testHeading: "Explicit",
-			variant: "explicit",
+			testHeading: "Default, with value and text overflow",
+			currentValue: "The selected value of the picker, with long text the triggers the overflow behavior with ellipsis",
 		},
 		{
-			testHeading: "Button",
-			variant: "button",
+			testHeading: "Quiet",
+			isQuiet: true,
+		},
+		{
+			testHeading: "Side label",
+			labelPosition: "side",
+		},
+		{
+			testHeading: "Side label, alignment with switch",
+			labelPosition: "side",
+			withSwitch: true,
+		},
+		{
+			testHeading: "With thumbnail icon",
+			showWorkflowIcon: true,
 		},
 	],
 	stateData: [
+		{
+			testHeading: "Hovered",
+			isHovered: true,
+		},
+		{
+			testHeading: "Active",
+			isActive: true,
+		},
+		{
+			testHeading: "Keyboard focused",
```

Comment:
Is there a focused test case at all, in addition to keyboard focused? I don't see it in the controls, and just quickly scanning the CSS, it looks like it's all been converted to using `:focus-visible`.

Reply:
There are no `:focus` styles, and all `:focus-visible` selectors are also combined with `.is-keyboardFocused`, so those styles should be covered.